src/test/java/kitchenpos/menus/tobe/application/TobeMenuGroupServiceTest.kt

```diff
@@ -0,0 +1,54 @@
+package kitchenpos.menus.tobe.application
+
+import kitchenpos.menus.tobe.domain.menugroup.TobeMenuGroup
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.transaction.annotation.Transactional
+import java.util.*
+
+@Transactional
+@SpringBootTest
+class TobeMenuGroupServiceTest
+    @Autowired
+    constructor(
+        private val sut: TobeMenuGroupService,
+    ) {
```

Comment:
```suggestion
class TobeMenuGroupServiceTest(
  @Autowired
  private val sut: TobeMenuGroupService,
) {
```

`@TestConstructor(autowireMode = TestConstructor.AutowireMode.ALL)` 을 사용해
혹은 생성자 주입을 사용해도 괜찮을 것 같아요. 

```suggestion
@TestConstructor(autowireMode = TestConstructor.AutowireMode.ALL)
class TobeMenuGroupServiceTest(
  private val sut: TobeMenuGroupService,
) {
```